src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -32,38 +18,6 @@ internal static partial class CURL_SEEKFUNC
             internal const int CURL_SEEKFUNC_CANTSEEK = 2;
         }
 
-        // Class for constants defined for the enum CURLMSG in multi.h
-        internal static partial class CURLMSG
-        {
-            internal const int CURLMSG_DONE = 1;
-        }
-
-        // Type definition of CURLMsg from multi.h
-        [StructLayout(LayoutKind.Explicit)]
-        internal struct CURLMsg
-        {
-            [FieldOffset(0)]
-            internal int msg;
-            [FieldOffset(8)]
-            internal IntPtr easy_handle;
-            [FieldOffset(16)]
-            internal IntPtr data;
-            [FieldOffset(16)]
-            internal Http.CURLcode result;
-        }
-
-        // Poll values used with curl_multi_wait and curl_waitfd.events/revents
-        internal const int CURL_WAIT_POLLIN = 0x0001;
-
-#pragma warning disable 0649 // until this file is split up, this produces a warning in the X509 project due to being unused
-        internal struct curl_waitfd
-        {
-            internal int fd;
-            internal short events;
-            internal short revents;
-        };
-#pragma warning restore 0649
-
         public delegate size_t curl_readwrite_callback(
             IntPtr buffer,
             size_t size,
```

Comment:
We shouldn't be using size_t in our interop.  But I guess this will be shim'd separately?